Articles of Governance/Battalion Standardization.md

```diff
@@ -1,11 +1,11 @@
-# Battalion Standardization Act v1.1
+# Battalion Standardization
 The battalion is the core organizational unit of the April Knights. They are the tools that the Grandmaster uses for planning, and their quality is of the utmost importance for the collective success of the Order. This act is to set maximums on the amount of members a battalion can have and how many Officers should be in each battalion.
 
 ### Section I
 A battalion is not to exceed one hundred (100) members. The make up of a battalion will consist of one (1) Commander, up to nine (9) Officers, and up to ninety (90) Oathsworn Knights, inclusive of any Corporals present among the Battalion’s ranks.
 
 ### Section II
-Among the nine (9) Officers, one will be designated the “First Officer.” The First Officer is responsible for casting the battalion vote in the Lower Council, if the Commander sits on the Upper Council. As such, the First Officer can be given to a Sergeant even if higher ranked Officers are present, as best suits the needs of the battalion, and at the Commander’s discretion. The First Officer has no responsibilities in the Upper Council, and as such, their main priority is the management and monitoring of their battalion. While the final authority rests with the commander, the First Officer is empowered to make operational calls when the commander is otherwise occupied. 
+Among the nine (9) Officers, one will be designated the “First Officer.” The First Officer is responsible for casting the battalion vote in the Lower Council, if the Commander sits on the Upper Council. As such, the First Officer can be given to a Sergeant even if higher ranked Officers are present, as best suits the needs of the battalion, and at the Commander’s discretion. The First Officer has no responsibilities in the Upper Council, and as such, their main priority is the management and monitoring of their battalion. While the final authority rests with the commander, the First Officer is empowered to make operational calls when the commander is otherwise occupied.
 
 ### Section III
 All battalions fall under the supremacy of the Grandmaster and the authority of the Councils, of which all Commanders are a member. Commanders who fail to recognize the supremacy and authority of either the Grandmaster or both of the Councils are guilty of insubordination and should be removed from their position of command.
```

Comment:
Section III doesn't specify how commanders should be removed from their position. Shouldn't it at least suggest something?